packages/nimble/R/cppDefs_core.R

```diff
@@ -7,13 +7,11 @@
 ## Base class for C++ file and compilation information
 cppDefinition <- setRefClass('cppDefinition', 
                              fields = list(
-                                 filename = 'ANY',	#  character, what filename (to which .h and .cpp will be appended) is this definition in
-                                 CPPusings = 'ANY',	#  character
```

Comment:
That makes a lot of sense!  We determined in the past that type-checking during routine use imposes very noticeable cost, but your suggestion would allow us to avoid a lot of splintered error-trapping during development.  Let's try it.